website/vercel.json

```diff
@@ -1,5 +1,75 @@
 {
   "cleanUrls": true,
+  "redirects": [
+    {
+      "source": "/live-updates",
+      "destination": "/what-are-live-updates"
+    },
+    {
+      "source": "/upgrade-guides",
+      "destination": "/for-iOS/upgrade-guides"
```

Comment:
Why is iOS capitalized while all other proper nouns (Android, Capacitor, React Native, etc.) are lower case? We typically use lower case in URLs.

Reply:
likely a copy paste thing. No real reason for it to be capitalized.